activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectEntity.java

```diff
@@ -53,6 +54,9 @@ public class ProjectEntity extends AuditableEntity<String> implements Project<St
 
     private String name;
 
+    @Column(name = "disp_name", nullable = false)
```

Comment:
For now the display name should be nullable (at least until it is also developed on FE side).

Reply:
Yes, done.